src/PublisherInterface.ts

```diff
@@ -546,3 +548,5 @@ export class PublisherConnector {
     }
   }
 }
+
+export {PublisherInterface as PublisherConnector};
```

Comment:
What is the reasoning here? Why keep the name on this instance?

Reply:
Just for backwards compatibility as we have a few clients running this in production. Again there are two view points on this, which is reflected in https://github.com/chili-publish/publisher-interface/issues/11 as well.

- One view point is we try to keep the Publisher Interface as backwards compatible as possible, as per my suggest in https://github.com/chili-publish/publisher-interface/issues/11 
- The other view point, is deprecate and just force a change, as you suggested in https://github.com/chili-publish/publisher-interface/issues/11 

I do not think either view point is wrong, but lets get a consensus in https://github.com/chili-publish/publisher-interface/issues/11 about this as well.

Reply:
See commit 7, I removed the alias and related docs until a decision is made